.gitignore

```diff
@@ -79,3 +79,7 @@ out.txt
 
 # ignore vscode delve debug binary
 */__debug_bin
+
+.run/
+/operator/config/crd/bases/component.kyma-project.io_kcpmodules.yaml
```

Comment:
why is this in gitignore?

Reply:
it's because that `make manifests` will scan all folders, and generate crds under this `crd/bases/` folder, but actually this kcpmodule is just for testing purpose (generated by `make test-crd`). I haven't find a good way to prevent this happen, so just add it to gitignore..